proto/regen/ecocredit/basket/v1/state.proto

```diff
@@ -86,3 +86,19 @@ message BasketBalance {
   // to create an index which is used to remove the oldest credits first.
   google.protobuf.Timestamp batch_start_date = 4;
 }
+
+// BasketFee is a valid coin denom and amount that may be used as the fee
+// for basket creation.
+message BasketFee {
+  // This table is controlled via governance.
+  option (cosmos.orm.v1alpha1.table) = {
+    id : 13,
+    primary_key : {fields : "denom"}
+  };
+
+  // denom is the denom of the fee required to create a basket.
+  string denom = 1;
+
+  // amount is the amount of the fee required to create a basket.
+  string amount = 2;
+}
```

Comment:
As mentioned in #960 in relation to the credit class fee, I think we only want to support a single fee. We named the parameter `basket_creation_fee` and copied the implementation for credit class fee but I don't actually recall any conversations about supporting multiple fees.

```suggestion
// BasketFee is a valid coin denom and amount that may be used as the fee
// for basket creation.
message BasketFee {
  // This table is controlled via governance.
  option (cosmos.orm.v1alpha1.singleton) = {
    id : 13,
    primary_key : {fields : "denom"}
  };

  // fee is the fee required to create a basket.
  cosmos.base.v1beta1.Coin fee = 1
}
```

Reply:
using Coin works as long as it's not the primary key, ORM can't handle imported messages as PK's. i think the denom/amount separation is necessary

Reply:
we don't need a primary key with a singleton. it just has one instance

Reply:
right - so then we aren't concerned with fees in other denoms? if we're just going with the amount of $regen only, we could just store the amount string

Reply:
We just want one fee (could be multiple coins). So just a singleton with repeated Coins